src/Ilios/CoreBundle/Tests/Entity/CurriculumInventoryInstitutionTest.php

```diff
@@ -22,6 +22,33 @@ protected function setUp()
         $this->object = new CurriculumInventoryInstitution;
     }
 
+    // public function testNotBlankValidation()
```

Comment:
We will need to change the CI Institution entity here to have consistency with address labeling..